packages/manager/modules/office/package.json

```diff
@@ -20,8 +20,12 @@
     "@ovh-ux/component-rollup-config": "^6.0.0"
   },
   "peerDependencies": {
-    "@ovh-ux/ovh-utils-angular": "^14.0.2",
+    "@ovh-ux/ng-ovh-swimming-poll": "^4.0.0",
+    "@ovh-ux/ng-ovh-utils": "^14.0.3",
+    "@ovh-ux/ng-ovh-web-universe-components": "^5.0.0",
     "angular": "^1.7.5",
-    "angular-ui-bootstrap": "1.3.3"
+    "angular-translate": "^2.18.1",
+    "angular-ui-bootstrap": "1.3.3",
+    "ovh-ui-angular": "^3.5.1"
```

Comment:
Following service is used `OvhHttp`. It means that `@ovh-ux/ng-ovh-http` should also be added as a peer dependency.